src/FormStyles.js

```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from "@material-ui/core";
```

Comment:
Will this file be used everywhere or is it specific to the register page?

Also, let's stick with the convention of naming react files uppercase and everything else lowercase.

Reply:
makeStyles will be used anywhere we need the form styles from Material UI. As of right now, it would apply to the register and sign-in pages.

Reply:
the naming convention of 'makeStyles' comes from material-ui, but I updated the react file and function 'formStyles' to 'FormStyles'. Hopefully, this is what you wanted.

Reply:
changed it back to 'formStyles' :)